src/reducers/auth.js

```diff
@@ -32,6 +35,18 @@ export default function (state = initialState, action) {
                 isLoggedIn: false,
                 user: null,
             };
+        case LOGIN_ATTEMPT_INIT:
```

Comment:
@hawaism no need to add this case as it automatically redirects to default.

Reply:
done